### PR TITLE
Change all username placeholders to username (#2568)

### PR DESCRIPTION
**Pull request by chase-metzger**
Fixes issue #2568

### Description

In the file `vi-github-and-repositories.md` there are multiple ways the original writers refer to the same GitHub username. That being the reader's. Rename placeholder for the reader's/vi's GitHub username from the inconsistent `<username>`, `username`, `YOUR_USERNAME` versions to `username`.

### Raw.Githack preview link

https://raw.githack.com/chase-metzger/chase-metzger.github.io/consistent-username-naming/pages/vi/vi-github-and-repositories.md

### CODE REVIEW

**Comment by Liew211** [comment]
@chase-metzger Looks like this is better off as `username` rather than `your_username`

File: pages/vi/vi-github-and-repositories.md [hunk 1]
```diff
@@ -120,7 +120,7 @@ aborts the commit.
...
+Now, your repository has been synced to the upstream/master. However, a discrepancy may still exist between your local (and now your origin/master) versus your username.github.io on GitHub. You will now use `git diff` and `git status` to check how your local repository compares to your username.github.io repository. Depending on whether you have more or less commits than your username.github.io, you will either use `git pull` to receive any changes or `git push` to push updates to your repository. Most likely, as you just synced with the master, you will use `git push` to push updates to username.github.io repo.
```
Comment (Liew211) [inline pages/vi/vi-github-and-repositories.md]: This should still say "your username.github.io repo" at the end

Comment (chase-metzger) [reply pages/vi/vi-github-and-repositories.md]: Fixed

**Comment by hiroTochigi** [comment]
@chase-metzger Raw.Githack preview link does not have  "#!". Therefore, your page does not render correctly. Please check the Githack section of [this page](https://open-learning-exchange.github.io/#!pages/vi/vi-github-issues.md) out

**Pull request merged**